lib/matplotlib/backends/backend_ps.py

```diff
@@ -1454,6 +1454,8 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     else: precmd = ''
     command = '%s cd "%s" && latex -interaction=nonstopmode "%s" > "%s"'\
                 %(precmd, tmpdir, latexfile, outfile)
+    # Replace ~ so Latex does not think it is line break
+    command=command.replace("~","\\string~")
```

Comment:
Can you still move that up and change `latexfile` and not `command`?

Squash (this assumes that you use git in the command line; if not, please tell... :-)):

* Make the changes
* add a new commit, `git add . & git commit -m "Fixup"`
* rebase-squash it all commits into only one: `git rebase -i origin/master` -> prepend all the commits but teh first one with `s` instead of `pick`, close the editor and let rebase do it's thing -> you will be prompted by an editor which shows all the commit messages combined -> edit it until you are happy
* force push: `git push -f`